cuBERT_topic_modelling/berttopic_example.ipynb

```diff
@@ -0,0 +1,617 @@
+{
```

Comment:
We need to fix the Explicitly set the environment variable TOKENIZERS_PARALLELISM=(true | false) .
You can set environment variables using something like below:

```python
import os
os.environ["TOKENIZERS_PARALLELISM"] = "true"
```